summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -183,6 +197,12 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
     }
 
     val countersForSpout: Seq[(Group, Name)] = JobCounters.getCountersForJob(jobID).getOrElse(Nil)
+    val isMergeableWithSource = getOrElse(stormDag, node, DEFAULT_FM_MERGEABLE_WITH_SOURCE).get
+    val sourceParallelism = getOrElse(stormDag, node, parOpt.getOrElse(DEFAULT_SOURCE_PARALLELISM)).parHint
+    val FMParallelism = getOrElse(stormDag, node, DEFAULT_FM_PARALLELISM).parHint
```

Comment:
Uppercase locals goes against style; I'd use flatMapParallelism here.

Also, rather than checking the `FlatMapParallelism` on the `Source` only, should we also check it on the `FlatMappedProducer`s as we're scanning above?